salt/utils/gitfs.py

```diff
@@ -510,11 +494,30 @@ def __init__(
             log.critical(msg, exc_info=True)
             failhard(self.role)
 
-    def full_id(self):
```

Comment:
We need to properly deprecate a function if its no longer needed.

Reply:
It was in the CVE fix

Reply:
If the rest of the team is okay with removing it then I am, but per policy it should properly be deprecated before removal.

Reply:
Remove it... it hasn't had time to have anything built around it...